unlock-app/src/components/content/certification/CertificationForm.tsx

```diff
@@ -0,0 +1,414 @@
+            </p>
+
+            <div className="grid grid-cols-2 gap-4">
+              <div>
+                <ImageUpload
+                  description="This illustration will be used for the NFT tickets. Use 512 by 512 pixels for best results."
+                  isUploading={isUploading}
+                  preview={metadataImage!}
+                  onChange={async (fileOrFileUrl: any) => {
+                    if (typeof fileOrFileUrl === 'string') {
+                      setValue('metadata.image', fileOrFileUrl)
+                    } else {
+                      const items = await uploadImage(fileOrFileUrl[0])
+                      const image = items?.[0]?.publicUrl
+                      if (!image) {
+                        return
+                      }
+                      setValue('metadata.image', image)
+                    }
+                  }}
+                />
+              </div>
+              <div className="grid gap-2">
+                <Input
+                  {...register('lock.name', {
+                    required: {
+                      value: true,
+                      message: 'Name is required',
+                    },
+                  })}
+                  type="text"
+                  placeholder="Name"
+                  label="Certification Name"
+                  description={
+                    'Enter program, course name to have recipient show off what they have completed.'
+                  }
+                  error={errors.lock?.name?.message}
+                />
+
+                <TextBox
+                  {...register('metadata.description', {
+                    required: {
+                      value: true,
+                      message: 'Please add a description for your certificate',
+                    },
+                  })}
+                  label="Description"
+                  placeholder="Write description here."
+                  description={<DescDescription />}
+                  rows={4}
+                  error={errors.metadata?.description?.message as string}
+                />
+
+                <Input
+                  {...register('metadata.certification.certification_issuer', {
+                    required: {
+                      value: true,
+                      message: 'Please add an issuer name',
+                    },
+                  })}
+                  label="Official Name of the Issuer"
+                  description="This is part of metadata to store the official name of issuer"
+                  error={
+                    // @ts-ignore
+                    errors.metadata?.certification?.issuer?.message as string
+                  }
+                />
+
+                <Input
+                  {...register('metadata.external_url')}
+                  label="External URL"
+                  description="Included a link in your certification, so members can learn about it through metadata"
```

Comment:
```suggestion
                  description="Include a link to the organization that performed the certification. This will be included in the NFT metadata."
```